dragnet/features.py

```diff
@@ -97,47 +97,47 @@ class CSSFeatures(object):
-                              'top',
-                              'content',
-                              'link',
-                              'title',
-                              'comment',
-                              'tools',
-                              'rating',
-                              'ss'],
-                     'class':['menu',
-                              'widget',
-                              'nav',
-                              'share',
-                              'facebook',
-                              'cat',
-                              'top',
-                              'content',
-                              'item',
-                              'twitter',
-                              'button',
-                              'title',
-                              'header',
-                              'ss',
-                              'post',
-                              'comment',
-                              'meta',
-                              'alt',
-                              'time',
-                              'depth',
-                              'thread',
-                              'author',
-                              'tools',
-                              'reply'
-                              'url',
-                              'avatar',
-                              'ss']}
+    attribute_tokens = {'id': ['nav',
+                               'ss',
+                               'top',
+                               'content',
+                               'link',
+                               'title',
+                               'comment',
+                               'tools',
+                               'rating',
+                               'ss'],
+                        'class': ['menu',
+                                  'widget',
+                                  'nav',
+                                  'share',
+                                  'facebook',
+                                  'cat',
+                                  'top',
+                                  'content',
+                                  'item',
+                                  'twitter',
+                                  'button',
+                                  'title',
+                                  'header',
+                                  'ss',
+                                  'post',
+                                  'comment',
+                                  'meta',
+                                  'alt',
+                                  'time',
+                                  'depth',
+                                  'thread',
+                                  'author',
+                                  'tools',
+                                  'reply',
+                                  'url',
+                                  'avatar']  # , 'ss']  <-- duplicate?!
```

Comment:
Is this the cause of the failing unittest?  These get encoded to 1-in-K features so if the dimensionality changes then this will cause things to fail (it's been a long time since I've looked at this).